### DISCUSSION TITLE
Pemula

### DISCUSSION THREAD

**Discussion by Satriabo** [community/community#19401] [Other Feature Feedback, Questions, & Ideas]
Saya ingin tau apakah akun githup saya sudah terferivikasi dan sudah aktif...?

**Comment by melvincwng** [comment]
Hi @Satriabo, from what I understood from Google translate, you are asking about whether your account is verified.

If that is the case, please check out this documentation link over [here](https://docs.github.com/en/get-started/signing-up-for-github/verifying-your-email-address) to determine whether your Github account is verified.

Hope this helps.

Terima kasih, selamat datang ke Komuniti Github :)
